multinet/__init__.py

```diff
@@ -41,6 +41,10 @@ def create_app(config: Optional[MutableMapping] = None) -> Flask:
     CORS(app, origins=allowed_origins, supports_credentials=True)
     Swagger(app, template_file="swagger/template.yaml")
 
+    # Set max file upload size to 2 MB
+    # TODO: decide on a non-arbitrary limit
+    app.config["MAX_CONTENT_LENGTH"] = 2000000
```

Comment:
32MB is a probably a good limit.

```suggestion
    app.config["MAX_CONTENT_LENGTH"] = 32 * 1024 * 1024
```